hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ErrorsAndExceptionsEnricher.java

```diff
@@ -2,10 +2,13 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Optional;
```

Comment:
What enrichers does `ErrorsAndExceptionsEnricher` depends on? Since, it now uses ApiTraceGraph, we want to make sure, it's running after the dependent enrichers have already run